aten/src/ATen/native/quantized/cpu/fused_obs_fake_quant.cpp

```diff
@@ -146,6 +145,14 @@ std::tuple<at::Tensor, at::Tensor> fused_moving_avg_obs_fake_quant_cpu(
     bool symmetric_quant) {
   // Calculate min/max
   auto observe = observer_on.item().toInt();
+  int64_t size = per_row_fake_quant ? self.size(0) : 1;
+  if (per_row_fake_quant && running_min.numel() == 0) {
+    float inf = std::numeric_limits<float>::infinity();
+    running_min.resize_(size).fill_(inf);
+    running_max.resize_(size).fill_(inf);
```

Comment:
n00b question, should this be `-inf` instead of `inf`?

Reply:
Yes, ideally. Although for the op it doesn't make a difference because we check for `std::isinf` which returns true either way.